File: C/DAY2/sumofnatural.c
#include<stdio.h>
int main(){

    //sum of n natural number(start from 1) given by user
    int n;
    int sum=0;
    printf("enter a natural number\n");
    scanf("%d",&n);

    for(int i=1;i<=n;i++){
         sum =sum + i; //or sum +=i;
 }
 
    printf("sum of number  is %d\n",sum);
    printf(" or using formula : %d",n*(n+1)/2);
    return 0;
}

File: EclipseWorkspace.java/CoreJavaInterview/src/javaInterview_10_7_25/SumofNatural.java
package javaInterview_10_7_25;

import java.util.Scanner;

public class SumofNatural {

	public static void main(String[] args) {
		
		int num,val=0;
		Scanner sc = new Scanner(System.in);
		System.out.println("enter a num : ");
		num = sc.nextInt();
		
		for(int i=1;i<=num;i++) {
//			val = i+val;
			val +=i;
			
		}
		
		System.out.println("the sum of n natural number: "+val);

		
	}
}
